client/src/cardComponent/cardComponent.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { Carousel, Card } from "react-bootstrap"
+import "./cardComponent.css"
+function cardComponent(props) {
+  return (
+    <div className="Society-cards">
+      <h1 className="heading">Societies under Geeks</h1>
+    <Carousel>
+      <Carousel.Item>
+        <Card style={{ width: '18rem' }}>
+          <Card.Body>
+            <Card.Title>Card Title</Card.Title>
+            <Card.Subtitle className="mb-2 text-muted">Card Subtitle</Card.Subtitle>
```

Comment:
Avoid using inline styling. CSS classes are generally better for performance than inline styles.

Reply:
That is inbuilt component of bootstrap.No changes made with inline styling from my end.